docs/release-notes/release-notes-0.18.0.md

```diff
@@ -118,20 +119,17 @@
 
 * [Fixed a bug in `btcd` that caused an incompatibility with
   `bitcoind v27.0`](https://github.com/lightningnetwork/lnd/pull/8573).
-  
-* [Fixed](https://github.com/lightningnetwork/lnd/pull/8609) a function call 
-  where arguments were swapped.
 
-* [Fixed](https://github.com/lightningnetwork/lnd/pull/8545) utxo selection
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8545) UTXO selection
   for the internal channel funding flow (Single and Batch Funding Flow). Now
-  utxos which are unconfirmed and originated from the sweeper subsystem are not
+  UTXOs which are unconfirmed and originated from the sweeper subsystem are not
   selected because they bear the risk of being replaced (BIP 125 RBF).
 
-* [Fixed](https://github.com/lightningnetwork/lnd/pull/8685) lncli "bumpfee"
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8685) `lncli bumpfee`
```

Comment:
nit: feel this should be removed since this bug was introduced in this release and later fixed.

Reply:
Done!